rel/overlay/etc/default.ini

```diff
@@ -280,13 +287,11 @@ port = 6984
 [couch_httpd_auth]
 ; WARNING! This only affects the node-local port (5986 by default).
 ; You probably want the settings under [chttpd].
-authentication_db = _users
-authentication_redirect = /_utils/session.html
-require_valid_user = false
-timeout = 600 ; number of seconds before automatic logout
-auth_cache_size = 50 ; size is number of cache entries
-allow_persistent_cookies = true ; set to false to disallow persistent cookies
-iterations = 10 ; iterations for password hashing
+
+; These settings were moved to [chttpd_auth]
+; authentication_db, authentication_redirect, require_valid_user,
+; timeout, auth_cache_size, allow_persistent_cookies, iterations
+
```

Comment:
I think we can move `min_iterations`, `max_iterations`, `password_scheme` and the rest of the settings.

Reply:
Moved to [chttpd].